portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,21 +30,31 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data") //"data" kept loading a blank screen no matter what I did so this was my workaround
+@WebServlet("/data") 
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> comments = new ArrayList<String>(
-    Arrays.asList("This site is awesome!!!!!!1!",
-    "Hi, I'm Andrew Yang, and I approve this message.",
-    "Hi, I'm Andrew Ying, and I dissaprove of the above.")
-  );
+  private ArrayList<Comment> comments = new ArrayList<Comment>(
+    Arrays.asList(new Comment("This site is awesome!!!!!!1!","Some fanboy"),
+    new Comment("Hi, I'm Andrew Yang, and I approve this message.","Andrew Yang"),
+    new Comment("Hi, I'm Andrew Ying, and I dissaprove of the above.","Andrew Ying")));
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    //create comment adn convert to Json
+    //create comment and convert to Json
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
Define these magic strings as constants at the top of the file or in their own constants class

Reply:
Good idea!